src/current/cockroachcloud/costs.md

```diff
@@ -163,17 +177,29 @@ Customers will not be charged for data transfer for CockroachDB {{ site.data.pro
 
 For CockroachDB Standard and Advanced, CockroachDB Cloud bills directly to customers based on their usage with a zero-dollar rate during the preview period.
 
-The usage data is based on the volume of data moved (per GiB) and the cloud provider’s line item for each type of data transfer, which CockroachDB categorizes as **Same-region data transfer**, **Cross-Region data transfer**, or **Public Internet data transfer**.
+The usage data is based on the volume of data moved (per GiB) and the cloud provider’s identified data transfer type, which CockroachDB categorizes as **Same-region data transfer**, **Cross-Region data transfer**, or **Public Internet data transfer**. Each category is reflected as a line item on your invoice.
 
 Cloud providers may vary in how they calculate each category. We recommend consulting the provider’s documentation for details (i.e. [AWS](https://aws.amazon.com/ec2/pricing/on-demand/), [GCP](https://cloud.google.com/vpc/network-pricing), or [Azure](https://azure.microsoft.com/en-us/pricing/details/bandwidth/)).
 
 #### Same-region
 
-A minimum production grade CockroachDB Advanced deployment consists of three nodes or more placed across three Availability Zones (AZs) within the selected region. When data moves between these AZs within a single region, this is considered same-region data transfer.
+There are no charges for same-region data transfer between CockroachDB cluster nodes. Data egressing the cluster to endpoints in the same region for backups or changefeeds, or using private connectivity, is charged the cloud provider's list price for same-region data transfer.
 
 #### Cross-region
 
-Similar to same-region data transfer , when data moves between CockroachDB nodes across multiple regions, we get cross-region charges (also known as inter-region charges). Typically, data transfer costs across regions are higher than data transfer costs across AZs (data centers) in a single region.
+When data moves between CockroachDB nodes that are in different regions, or from a CockroachDB node to another endpoint in a different region, it is metered as cross-region data transfer. 
+
+Cross-region data transfer includes:
+
+- Data transfer required to support queries that involve lookups on nodes in another region.  
```

Comment:
This has a lot of indirection and could be rewritten a bit to avoid "required to support" and "involve". Maybe "Queries that read or write data on a node in a different region" maybe?